### PR TITLE
[FIX] change schema name to aggregate data 

### PR DESCRIPTION
**Pull request by VanshIntel**
gpu_metrics_sampler

### CODE REVIEW

**Comment by tom95858** [comment]
@morrone Did you review this and find it OK?

**Comment by morrone** [comment]
> @morrone Did you review this and find it OK?

It doesn't have my name on it anymore, and looks like a stand-alone commit. That is good. But the commit message doesn't really meet the minimal bar for acceptance.

I don't have any comment on the actual code change.

**Review by tom95858 — CHANGES_REQUESTED** [review]
The commit message does not reflect what is actually being done here. Could you please fixup the metric names are requested and rewrite the commit message?

File: ldms/src/contrib/sampler/gpu_metrics_sampler/gmg_ldms_util.c [hunk 1]
```diff
@@ -81,7 +81,7 @@ const metric_t metricsDefinitions[] = {
         {.name = "perf_level", .type = LDMS_V_D64, .pf = (funcPtr_t) getPerfLevel},
         {.name = "power_usage (mW)", .type = LDMS_V_S32, .pf = (funcPtr_t) getPowerUsage},
 //        {.name = "power_cap (mW)", .type = LDMS_V_S32, .pf = (funcPtr_t) getPowerCap},    // no longer supported
-        {.name = "gpu_temp (Celsius)", .type = LDMS_V_D64, .pf = (funcPtr_t) getGpuTemp},
+        {.name = "gpu_temp (Celsius)", .type = LDMS_V_D64, .pf = (funcPtr_t) getGpuTemp}
```
[the remark this answers is not in the document]
Comment (VanshIntel) [reply ldms/src/contrib/sampler/gpu_metrics_sampler/gmg_ldms_util.c]: @tom95858 I removed the Unit from metric name, Please let me know if any other changes require.

File: ldms/src/contrib/sampler/gpu_metrics_sampler/gmg_ldms_util.c [hunk 2]
```diff
@@ -81,7 +81,7 @@ const metric_t metricsDefinitions[] = {
         {.name = "perf_level", .type = LDMS_V_D64, .pf = (funcPtr_t) getPerfLevel},
         {.name = "power_usage (mW)", .type = LDMS_V_S32, .pf = (funcPtr_t) getPowerUsage},
```
Comment (tom95858) [inline ldms/src/contrib/sampler/gpu_metrics_sampler/gmg_ldms_util.c:82]: @VanshIntel it looks like almost all of these metric names have the unit string in them. What I'm asking is that you please:
-  remove the '(\<unit-name\>)' from _all_ of the metric names
- add the unit string you remove to an entry called .unit="<unit>". Don't include the '()' in that string

For example,

```
   . . .
  {.name = "power_usage", .unit="mW", .type = LDMS_V_S32, .pf = (funcPtr_t)getPowerUsage},
  {.name = "gpu_temp", .unit="Celsius", .type = LDMS_V_D64, .pf = (funcPtr_t) getGpuTemp},
   . . .
```

The other change I'm asking that you make is to change the commit message to something like:

"Modify the schema name and add the set metric unit names"

Comment (VanshIntel) [reply ldms/src/contrib/sampler/gpu_metrics_sampler/gmg_ldms_util.c]: @tom95858 Thanks for the pointing the issue, I've made the changes accordingly. Please let me know if there is any other changes require.

Comment (tom95858) [reply ldms/src/contrib/sampler/gpu_metrics_sampler/gmg_ldms_util.c]: Thanks @VanshIntel. Could you please change the commit message to something like:

```
Refactor schema and metric names.

The schema name is changed to "gpumetrics"

The metric names have been refactored to move the unit
string from the metric name to the .unit field of the schema
metric data structure.
```

Comment (tom95858) [reply ldms/src/contrib/sampler/gpu_metrics_sampler/gmg_ldms_util.c]: @VanshIntel also please squash all of this into a single commit.

Comment (VanshIntel) [reply ldms/src/contrib/sampler/gpu_metrics_sampler/gmg_ldms_util.c]: @tom95858 I've merge all the changes into single commit. Please let me know if it require any changes.

Comment (VanshIntel) [reply ldms/src/contrib/sampler/gpu_metrics_sampler/gmg_ldms_util.c]: @tom95858 Sorry for the confusion. I've just updated the comment. Please let me know if there is any changes require. Thanks

Comment (VanshIntel) [reply ldms/src/contrib/sampler/gpu_metrics_sampler/gmg_ldms_util.c]: @tom95858 I've pushed the changes. Please let me know if you require any changes.

Comment (VanshIntel) [reply ldms/src/contrib/sampler/gpu_metrics_sampler/gmg_ldms_util.c]: @tom95858 Can you please approve this merge or Please let me know if any changes require.

**Comment by VanshIntel** [comment]
@tom95858  Can you please review this merge or Please let me know if any changes require.

**Pull request closed without merging**